docs/network_architecture.rst

```diff
@@ -347,27 +415,21 @@ where the request and response data are given as the following protobuf messages
 
    message StartTrainingRoundRequest {}
 
+
    message StartTrainingRoundResponse {
-       xain_proto.np.NDArray weights = 1;
-       int32 epochs = 2;
-       int32 epoch_base = 3;
+       int32 epochs = 1;
+       int32 epoch_base = 2;
    }
 
    message EndTrainingRoundRequest {
-       xain_proto.np.NDArray weights = 1;
+       string participant_id = 1;
        int32 number_samples = 2;
        map<string, xain_proto.np.NDArray> metrics = 3;
```

Comment:
not directly related to this mr, but while you are on it, can you please change this line to `string metrics = 3;` which was somehow forgotten earlier on?